spec/requests/api/v1/course_spec.rb

```diff
@@ -552,5 +494,62 @@ def get_next_courses(link, params = {})
         expect(Subject.where(subject_code: "U3").count).to eq(0)
       end
     end
+
+    describe "age_range" do
```

Comment:
These specs needed to be extracted to their own describe block as different course setups are required (primary course and secondary course).